.travis.yml

```diff
@@ -68,3 +70,15 @@ script:
   - set -e; python -c "import ecl"; set +e
   - ctest --output-on-failure $TEST_SUITE
   - popd
+
+deploy:
+    - provider: pypi # source distribution (done linux python 2.7 only)
+      skip_cleanup: true
+      skip_upload_docs: true
+      user: xjules
+      distributions: sdist bdist_wheel
+      password:
+        secure: qqN8VyzV4YbdUofCoo65KWO+87+1PcOHpKp4JKHXwZ9uandFjwdvuTYiZ/XR4yk0rLa28qpa9iUfk4iWkC8zfFcKWuJ/9I45I8kbhvM6tD2UtCGfmR3AzNceEC1/9OBCpnBU8MTTrY9SNULeqanOPPQlvy8X54lAdEbXHOzlNYheAG+htE8PSUO0FTOaYrDAfR+slRBSQ8OqDuhi/MNo87az85jmX9ONbkuaOyoc13QoVlQ+Yzfw0eBQjzg+C+59JGNp18Fpoyp4NDSYyE1LRUbNMr1ud7id8AIg9tMOc1pjYk8FBIIT2hnSKh1BdQm2lLvQRQWIznsspDMrey/AXOf8knPqR1TjVxVTEbjdvDtQHibIA2vAyX6s4vYBbZfL/jelv11IWS02lgjzJCKXzvkJUsSymmVCHF//zOOjJfZoEJOIzPH2jwwd1iWuaeE4C6JIpU2QV3x8gsNPJjesJgAr7je25ciDOIlyeNfoubIl713Q3BKyAiQBk3d6bpd0o8p57rOoWegoYSe7sfPGyrM88UlbGvmM7Rj+DJHl7jMy711BIf6R51lJguwVgzRg6r+jRdAHLKEpZZ7oc33BVEo4ZowU2awywepo/IVNTFi6X2S24oZ/ZalRLKM+W3yRPRZXifgfqW2Ssk0qK+7SlBmpDYteXuThyFjpBUWilbA=
```

Comment:
I guess this is a token that can be used to upload stuff to pypi. Libecl is a public repo, are we sure we want to leave this thing here? I mean, people, could have their own build where they inject malicious code and use this token to upload it to pypi without us noticing ...

Maybe we should have a private repo with just the .travis file for building and deploying ecl stable

@jokva @pgdr I know you guys worked on something like this before, what do you think?

Reply:
A good point. @jokva let's have a joint discussion about having a separate deploy repo.

Reply:
After the morning's discussion we will decide whether we really need pip install now or we can wait until ecl3.0.

Reply:
@andreabrambilla the token is encrypted by Travis and cannot be used in other repos.